cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -325,101 +301,18 @@ void CobTwistController::twistCallback(const geometry_msgs::Twist::ConstPtr& msg
 void CobTwistController::solveTwist(KDL::Twist twist)
 {
-    geometry_msgs::Pose pose_tip, pose_base;
-    geometry_msgs::Point point_base, point_ee;
-
     int ret_ik;
     KDL::JntArray q_dot_ik(chain_.getNrOfJoints());
 
-    if(twist_controller_params_.base_active)
-    {
-        #if DEBUG == 1
-            debug();
-            if(!reset_markers_){
-                point_ee.x = odom_transform_ct.getOrigin().x();
-                point_ee.y = odom_transform_ct.getOrigin().y();
-                point_ee.z = odom_transform_ct.getOrigin().z();
-
-                point_base.x = odom_transform_bl.getOrigin().x();
-                point_base.y = odom_transform_bl.getOrigin().y();
-                point_base.z = odom_transform_bl.getOrigin().z();
-
-                point_ee_vec_.push_back(point_ee);
-                point_base_vec_.push_back(point_base);
-
-                double id1 = 0;
-                double id2 = 1;
-
-                showMarker(id1,1,0,0,"m",debug_base_compensation_visual_tip_pub_,point_ee_vec_);
-                showMarker(id2,0,1,0,"m",debug_base_compensation_visual_base_pub_,point_base_vec_);
-            }
-
-            if(reset_markers_){
-                point_ee_vec_.clear();
-                point_base_vec_.clear();
-            }
-        #endif
-        q_dot_ik.resize(chain_.getNrOfJoints() + 3); // + 3 for base
-        try
-        {
-            tf_listener_.waitForTransform("base_link",chain_tip_link_, ros::Time(0), ros::Duration(0.5));
-            tf_listener_.lookupTransform("base_link",chain_tip_link_,  ros::Time(0), bl_transform_ct);
-        }
-        catch (tf::TransformException &ex)
-        {
-            ROS_ERROR("%s",ex.what());
-            return;
-        }
-
-        bl_frame_ct.p = KDL::Vector(bl_transform_ct.getOrigin().x(), bl_transform_ct.getOrigin().y(), bl_transform_ct.getOrigin().z());
-        bl_frame_ct.M = KDL::Rotation::Quaternion(bl_transform_ct.getRotation().x(), bl_transform_ct.getRotation().y(), bl_transform_ct.getRotation().z(), bl_transform_ct.getRotation().w());
-
-        try
-        {
-            tf_listener_.waitForTransform(chain_base_link_,"base_link", ros::Time(0), ros::Duration(0.5));
-            tf_listener_.lookupTransform(chain_base_link_,"base_link", ros::Time(0), cb_transform_bl);
-        }
-        catch (tf::TransformException &ex)
-        {
-            ROS_ERROR("%s",ex.what());
-            return;
-        }
-
-        cb_frame_bl.p = KDL::Vector(cb_transform_bl.getOrigin().x(), cb_transform_bl.getOrigin().y(), cb_transform_bl.getOrigin().z());
-        cb_frame_bl.M = KDL::Rotation::Quaternion(cb_transform_bl.getRotation().x(), cb_transform_bl.getRotation().y(), cb_transform_bl.getRotation().z(), cb_transform_bl.getRotation().w());
-
-        //Solve twist
-        ret_ik = p_inv_diff_kin_solver_->CartToJnt(last_q_, last_q_dot_, twist, bl_frame_ct, cb_frame_bl, q_dot_ik);
-    }
```

Comment:
this block has moved into adjust_jac() in kinematics_extenison.cpp

ToDo: except for the debug part...